src/servers/src/prometheus.rs

```diff
@@ -92,6 +92,7 @@ impl PrometheusServer {
 
     pub fn make_app(&self) -> Router {
```

Comment:
for now, what we do about this method? it seems is no longer used anymore.

Reply:
It depends on whether we still need it to be a dedicate port. I haven't figured it out, do you have any thought? cc @sunng87 @evenyag. Or we can make it configurable?

Reply:
It depends on the acceptance of Prometheus ecosystem for url context path. I have verified that Grafana has great support for url prefix (it can even parse query string from url and append it to actual query). I also checked open source clients in [java](https://github.com/mckang/promql_java_client/blob/master/promql_client/src/main/java/com/bdwise/prometheus/client/builder/RangeQueryBuilder.java#L23) and [python](https://github.com/4n4nd/prometheus-api-client-python/blob/master/prometheus_api_client/prometheus_connect.py#L121), both works with context path.

Do you know any client that doesn't work with context path?

In contrast, influxdb ecosystem has bad support for context path which is a road blocker for our influxdb api.

Reply:
Great, then we can remote this port and all related logic 👍

Reply:
the all prometheus server relevant logic has been deleted